code/src/components/GameWrapper.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+
+import { useSelector } from 'react-redux'
+import { Game } from './Game'
+import { CreateUsername } from './CreateUsername'
+
+export const GameWrapper = () => {
+  const username = useSelector(state => state.rooms.username)
+
+  return (
+    <div className="game-container">
+      {/* If the username hasn't been set, show "CreateUsername" component */}
+      {
+        username === "" &&
+        <CreateUsername />
+      }
+      {
+        // If username has been set, go to "Game".
+        username !== "" &&
+        <Game />
+      }
```

Comment:
Nice to see how you've done another conditional statement using the ternary operators. Makes sense and easy to follow that if the user hasn't inputted username then the input field and button is show otherwise they will go into the game!